### PR TITLE
mr-60-product-vertical-not-aligned

### PR DESCRIPTION
**Pull request by JaqKent**
fixes in image not aligned and font sizes and weight

### CODE REVIEW

File: src/components/CustomCard/index.tsx [hunk 1]
```diff
@@ -53,13 +53,34 @@ function CustomCard({
...
+                        <div>
+                            <span
+                                className={
+                                    small
+                                        ? styles.smallRating
+                                        : styles.largeRating
+                                }
+                            >
+                                <FontAwesomeIcon icon={faStar} />
+                                {stars}{' '}
```
Comment (nathan-hyan) [inline src/components/CustomCard/index.tsx:67]: Remove extra space `{' '}`

File: src/components/CustomCard/index.tsx [hunk 2]
```diff
@@ -53,13 +53,34 @@ function CustomCard({
...
+                            </span>
+                            <span
+                                className={
+                                    small
+                                        ? styles.smallVotes
+                                        : styles.largeVotes
+                                }
+                            >
+                                ({votes})
+                            </span>{' '}
```
Comment (nathan-hyan) [inline src/components/CustomCard/index.tsx:77]: same here

File: src/components/CustomCard/styles.module.scss [hunk 3]
```diff
@@ -106,45 +109,52 @@
...
-    font-weight: bold;
-    font-size: 1.2rem;
+    @include flex(space-between, flex-start, column);
 }
 
 .largeSection {
     @include flex(center, flex-start, row);
     padding-top: .5rem;
+}
+
```
Comment (nathan-hyan) [inline src/components/CustomCard/styles.module.scss:151]: Remove extra space

**Review by nathan-hyan — COMMENTED** [review]
Great work! Some comments tho

File: src/components/CustomCard/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/CustomCard/index.tsx:67]: done

File: src/components/CustomCard/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/CustomCard/index.tsx:77]: done

File: src/components/CustomCard/styles.module.scss (hunk 3 above)
Comment (JaqKent) [reply src/components/CustomCard/styles.module.scss:151]: done

**Pull request merged**